models/login_source.go

```diff
@@ -133,6 +133,7 @@ type LoginSource struct {
 	Type      LoginType
 	Name      string          `xorm:"UNIQUE"`
 	IsActived bool            `xorm:"NOT NULL DEFAULT false"`
+	IsDefault bool            `xorm:"DEFAULT NULL"` // for backward compatible
```

Comment:
s/DEFAULT NULL/DEFAULT false/

and no need for extra comment

Reply:
sure